<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-02T19:17:57 MDT" name="Default suite" finished-at="2022-06-02T19:18:34 MDT" duration-ms="36967">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.FailedTestRunner@248e319b]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2022-06-02T19:17:57 MDT" name="Default test" finished-at="2022-06-02T19:18:34 MDT" duration-ms="36967">
      <class name="testRunner.FailedTestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:testRunner.FailedTestRunner@248e319b]" started-at="2022-06-02T19:17:57 MDT" name="setUpClass" finished-at="2022-06-02T19:17:58 MDT" duration-ms="545" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@5203c80f]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.FailedTestRunner@248e319b]" started-at="2022-06-02T19:17:58 MDT" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-06-02T19:18:17 MDT" duration-ms="18940" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Verify product in offerpage"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Verifying Product in OfferPage"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [null] but found [Tomato]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [null] but found [Tomato]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at stepDefinitions.OfferStepDef.user_check_if_the_product_exists(OfferStepDef.java:68)
at ✽.User check if the product exists Tom(file:///C:/Users/nsuma/OneDrive/Desktop/Selenium%20Workspace/Cucumber/src/test/java/Feature/Offer.feature:12)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.FailedTestRunner@248e319b]" started-at="2022-06-02T19:18:17 MDT" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-06-02T19:18:34 MDT" duration-ms="16968" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Verify product in offerpage"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Verifying Product in OfferPage"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [null] but found [No data]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [null] but found [No data]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at stepDefinitions.OfferStepDef.user_check_if_the_product_exists(OfferStepDef.java:68)
at ✽.User check if the product exists Beet(file:///C:/Users/nsuma/OneDrive/Desktop/Selenium%20Workspace/Cucumber/src/test/java/Feature/Offer.feature:12)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:testRunner.FailedTestRunner@248e319b]" started-at="2022-06-02T19:18:34 MDT" name="tearDownClass" finished-at="2022-06-02T19:18:34 MDT" duration-ms="475" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- testRunner.FailedTestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
